### DISCUSSION TITLE
Physical i stall

### DISCUSSION THREAD

**Discussion by wolffman122** [Dilbert66/esphome-vistaECP#70] [Q&A]
Hi i have the system built on a bread board.  When i got to actually install in the alarm box, should i kill the ac and battery power to the alarm panel?

**Comment by Dilbert66** [comment]
I would recommended that yes you power off the system for safefty, but I have connected it a few times while still running though I don't think that's a good idea. Do as I say and not as I do :)

**Reply by wolffman122** [reply]
So I hooked up the NodeMCU and nothing blew up few.  The strange thing is now the NodeMCU is barely connecting to my wifi.  I have even tried a second NodeMCU same result.  If I pull the NodeMCU out of the bread board and leave it in the same spot the wifi connection is fine.  Any ideas?

I built the third type of circuit with the three isolators.

**Comment by Dilbert66** [comment]
I would suspect that you have some type of wiring issue somewhere in your circuit.

**Reply by wolffman122** [reply]
Is it ok to be powering the whole circuit through the nodemcu or should i provide separate 3.3 power supply?

**Reply by Dilbert66** [reply]
It's not a problem.
